ARTEMIS-5495: jakarta.servlet-api

GChuf opened the pull request:
Bumping jakarta.servlet-api.version to latest version (compiled with java 11 as opposed to 1.8).

gemmellr commented:
As dependency updates should in general, and especially HawtIO as the key dep of the console, these should clearly have Jiras. They would also typically be in their own changes.

That said, given Artemis Jetty usage specifically targets ee9 and the original HawtIO bits do also, I don't see why we would update the API usage to 6.x at this time.

GChuf commented:
@gemmellr thanks - i split the PR and added jiras.
As for artemis, I see that it uses ee9, but I see no references to it in hawtio. Given that artemis and hawtio both use jetty v12 now, and jetty v12 supports ee10, I thought it was okay to update.

GChuf commented:
I understand and I agree.
Closing this PR since hawtio remains on v5.
